modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java

```diff
@@ -213,9 +232,20 @@ public void testBusinessOrganizationCommerceAccountsVisibility()
 				CommerceAccountConstants.SITE_TYPE_B2B, StringPool.BLANK, null,
 				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
 
+		int organizationUser1CommerceAccountsCount =
```

Comment:
Lets keep the pattern the same as you had below

```suggestion
		int organizationUserCommerceAccountsCount1 =
```